lib/pages/classes/view/class_view.dart

```diff
@@ -241,4 +230,67 @@ class _ClassViewState extends State<ClassView> {
       ),
     );
   }
+
+  Widget lecturerCard(BuildContext context) {
+    final personProvider = Provider.of<PersonProvider>(context);
+
+    return Card(
+      key: const Key('LecturerCard'),
+      child: Padding(
+        padding: const EdgeInsets.all(8),
+        child: Row(
+          children: [
+            ClassIcon(classHeader: widget.classHeader),
+            const SizedBox(width: 8),
+            Expanded(
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  IconText(
+                    icon: Icons.class_,
+                    text: widget.classHeader.name ?? '-',
+                    style: Theme.of(context).textTheme.bodyText1,
+                  ),
+                  FutureBuilder(
+                    future: personProvider
+                        .mostRecentLecturer(widget.classHeader.id),
+                    builder: (context, snapshot) {
+                      if (snapshot.connectionState == ConnectionState.done) {
+                        final data = snapshot.data;
```

Comment:
Nit: Maybe call this something more informative than just `data` - `lectuerName`?